packager.go

```diff
@@ -218,9 +218,9 @@ func NewPackager(cert *x509.Certificate, key crypto.PrivateKey, packagerBase str
 	if !acceptablePackagerSchemes[baseURL.Scheme] {
 		return nil, errors.Errorf("PackagerBase %q must be over http or https.", packagerBase)
 	}
-	validityURL, err := url.Parse("https://cdn.ampproject.org/null-validity")
+	validityURL, err := url.Parse(path.Join(packagerBase, ValidityMapURL))
```

Comment:
I think this has a bug: https://play.golang.org/p/UdOl82Y1BCL breaks the double-slash.

Reply:
Ah, yes, `path.Join` changes it to `http:/localhost/amppkg/validity` and `url.Parse` changes that to `http:///localhost/amppkg/validity`. Since `packagerBase` is guaranteed to have a trailing slash in `config.go`, using `+` for `packagerBase and `ValidityMapURL`.